packages/reports/tests/acceptance/navi-report-test.js

```diff
@@ -1653,6 +1653,116 @@ module('Acceptance | Navi Report', function(hooks) {
+    await click('.ember-power-calendar-day[data-date="2015-11-18"]');
+
+    assert.dom('.filter-values--date-range-input__low-value').hasText('Nov 18, 2015', 'The start date is Nov 18, 2015');
+    assert
+      .dom('.filter-values--date-range-input__high-value')
+      .hasText('Nov 15, 2015', 'The end date is still Nov 15, 2015');
+
+    //collapse filters
+    await click('.report-builder__container-header__filters-toggle');
+    assert
+      .dom('.filter-collection')
+      .hasText('Date Time (Day) between Nov 18, 2015 - Nov 15, 2015', 'Collapsed range is Nov 18, 2015 - Nov 15, 2015');
+    //expand filters
+    await click('.filter-collection--collapsed');
+
+    await click('.navi-report__run-btn');
+
+    assert
+      .dom('.navi-info-message__error-list')
+      .hasText(
+        'The start date should be before end date',
+        '"The start date should be before end date" error is rendered'
+      );
+
+    //start date is equal to end date
+    await clickTrigger('.filter-values--date-range-input__high-value');
+    await click('.ember-power-calendar-day[data-date="2015-11-18"]');
+
+    assert
+      .dom('.filter-values--date-range-input__low-value')
+      .hasText('Nov 18, 2015', 'The start date is still Nov 18, 2015');
+    assert.dom('.filter-values--date-range-input__high-value').hasText('Nov 18, 2015', 'The end date is Nov 18, 2015');
+
+    //collapse filters
+    await click('.report-builder__container-header__filters-toggle');
+    assert.dom('.filter-collection').hasText('Date Time (Day) between Nov 18, 2015', 'Collapsed range is Nov 18, 2015');
+    //expand filters
+    await click('.filter-collection--collapsed');
+
+    await click('.navi-report__run-btn');
+
+    assert.dom('.navi-info-message__error-list').doesNotExist('No error if dates are equal');
+
+    //start date 1 day beyond end date
+    await clickTrigger('.filter-values--date-range-input__low-value');
+    await click('.ember-power-calendar-day[data-date="2015-11-19"]');
+
+    assert.dom('.filter-values--date-range-input__low-value').hasText('Nov 19, 2015', 'The start date is Nov 19, 2015');
+    assert
+      .dom('.filter-values--date-range-input__high-value')
+      .hasText('Nov 18, 2015', 'The end date is still Nov 18, 2015');
+
+    //collapse filters
+    await click('.report-builder__container-header__filters-toggle');
+    assert
+      .dom('.filter-collection')
+      .hasText('Date Time (Day) between Nov 19, 2015 - Nov 18, 2015', 'Collapsed range is Nov 19, 2015 - Nov 18, 2015');
+    //expand filters
+    await click('.filter-collection--collapsed');
+
+    await click('.navi-report__run-btn');
+
+    assert
+      .dom('.navi-info-message__error-list')
+      .hasText(
+        'The start date should be before end date',
+        '"The start date should be before end date" error is rendered when start date is 1 day beyond end date'
+      );
+
+    //select month grain
+    await clickItem('timeGrain', 'Month');
```

Comment:
Maybe make a test for `week`, `year`, etc.

Also heads up depending on if my column config item PR gets merged in I moved the `navi-report-test` to it's own folder and made a another test file so we can start splitting it up by what's being tested